src/main/java/shop/service/ProductService.java

```diff
@@ -8,7 +8,9 @@
 
     void getProduct(String productId);
 
-    List<Product> getAllProducts();
+    void getProductByName (String productName);
```

Comment:
method void but u want to GET smth from service. Please change it to return Product